internal/manifests/manifestutils/annotations.go

```diff
@@ -27,15 +27,6 @@ func Annotations(instance v1beta1.OpenTelemetryCollector, filterAnnotations []st
 	// new map every time, so that we don't touch the instance's annotations
 	annotations := map[string]string{}
 
-	// Enable Prometheus annotations by default if DisablePrometheusAnnotations is nil or true
-	if !instance.Spec.Observability.Metrics.DisablePrometheusAnnotations {
-		// Set default Prometheus annotations
-		annotations["prometheus.io/scrape"] = "true"
-		annotations["prometheus.io/port"] = "8888"
-		annotations["prometheus.io/path"] = "/metrics"
-	}
-
-	// allow override of prometheus annotations
```

Comment:
Adding the ConfigMap Hash annotation should also be moved to `PodAnnotations`. We don't need that annotation anywhere else, and this should also reduce the amount of test changes.

Reply:
I somewhat agree with you about adding opentelemetry-operator-config/sha256 annotation only to PodAnnotations.
@jaronoff97 @swiatekm 
 can we agree on that?

Reply:
yes, i think that makes sense.

Reply:
@swiatekm I've changed it, please check again.